integration-tests/tests/errors/index.spec.js

```diff
@@ -0,0 +1,125 @@
+    await browser.assert.ok(!!errorSpan, 'Checking presence of error span.');
+
+    const tags = errorSpan.tags;
+    await browser.assert.strictEqual(tags['location.href'], browser.globals.getUrl('/errors/templates/js-syntax-error.ejs'));
+    await browser.assert.strictEqual(tags['splunk.rumVersion'], require(path.join(__dirname, '..', '..', '..', 'package.json')).version);
+    await browser.assert.strictEqual(tags['app'], 'splunk-otel-js-dummy-app');
+    await browser.assert.strictEqual(tags['component'], 'error');
+    await browser.assert.strictEqual(tags['error'], 'true');
+    await browser.assert.strictEqual(tags['error.object'], 'SyntaxError');
+
+    switch (browser.options.desiredCapabilities.browserName.toLowerCase()) {
+    case 'chrome':
+      await browser.assert.strictEqual(tags['error.message'], 'Unexpected token \';\'');
+      break;
+    case 'firefox':
+      await browser.assert.strictEqual(tags['error.message'], 'expected expression, got \';\'');
+      break;
+    case 'safari':
+      await browser.assert.strictEqual(tags['error.message'], 'Unexpected token \';\'');
+      break;
+    }
+
+    await browser.assert.strictEqual(tags['ot.status_code'], 'UNSET');
+
+    await browser.end();
+  },
+  'JS unhandled error': async function(browser) {
+    browser.globals.clearReceivedSpans();
+    const url = browser.globals.getUrl('/errors/templates/unhandled-error.ejs');
+    await browser.url(url);
+
+    const errorSpan = await browser.globals.findSpan(s => s.name === 'onerror');
+    await browser.assert.ok(!!errorSpan, 'Checking presence of error span.');
+
+    const tags = errorSpan.tags;
+    await browser.assert.strictEqual(tags['location.href'], browser.globals.getUrl('/errors/templates/unhandled-error.ejs'));
+    await browser.assert.strictEqual(tags['splunk.rumVersion'], require(path.join(__dirname, '..', '..', '..', 'package.json')).version);
+    await browser.assert.strictEqual(tags['app'], 'splunk-otel-js-dummy-app');
+    await browser.assert.strictEqual(tags['component'], 'error');
+    await browser.assert.strictEqual(tags['error'], 'true');
+    await browser.assert.strictEqual(tags['error.object'], 'TypeError');
+
+    switch (browser.options.desiredCapabilities.browserName.toLowerCase()) {
+    case 'chrome':
+      await browser.assert.strictEqual(tags['error.message'], 'Cannot set property \'prop1\' of null');
+      break;
+    case 'firefox':
+      await browser.assert.strictEqual(tags['error.message'], 'test is null');
+      break;
+    case 'safari':
+      await browser.assert.strictEqual(tags['error.message'], 'null is not an object (evaluating \'test.prop1 = true\')');
+      break;
+    }
+
+    await browser.assert.strictEqual(tags['ot.status_code'], 'UNSET');
+
+    await browser.end();
+  },
+  'unhandled promise rejection': async function(browser) {
+    browser.globals.clearReceivedSpans();
+    const url = browser.globals.getUrl('/errors/templates/unhandled-rejection.ejs');
```

Comment:
I like that each of these test pages are in separate files; it makes it cleaner.